kunta-api-server/src/test/java/fi/otavanopisto/kuntaapi/test/server/integrations/mikkelinyt/MikkeliNytRestTestsIT.java

```diff
@@ -0,0 +1,407 @@
+    insertOrganizationSetting(kuntaApiOrganizationId, MikkeliNytConsts.ORGANIZATION_SETTING_LOCATION, ORGANIZATION_SETTING_LOCATION);
+    insertOrganizationSetting(kuntaApiOrganizationId, MikkeliNytConsts.ORGANIZATION_SETTING_BASEURL, baseUrl);
+    insertOrganizationSetting(kuntaApiOrganizationId, MikkeliNytConsts.ORGANIZATION_SETTING_IMAGEBASEURL, String.format("%s/%s/1000/", baseUrl, imagesBasePath));
+  }
+  
+  private void deleteSettings(String kuntaApiOrganizationId) {
+    deleteSystemSetting(MikkeliNytConsts.SYSTEM_SETTING_APIKEY);
+    deleteOrganizationSetting(kuntaApiOrganizationId, MikkeliNytConsts.ORGANIZATION_SETTING_LOCATION);
+    deleteOrganizationSetting(kuntaApiOrganizationId, MikkeliNytConsts.ORGANIZATION_SETTING_BASEURL);
+    deleteOrganizationSetting(kuntaApiOrganizationId, MikkeliNytConsts.ORGANIZATION_SETTING_IMAGEBASEURL);
+  }
+  
+  private class MikkeliNytMocker extends AbstractMocker {
+    
+    private List<Event> events;
+    
+    public MikkeliNytMocker() {
+      this.events = new ArrayList<>();
+    }
+    
+    /**
+     * Mocks a MikkeliNyt event
+     * 
+     * @param kuntaApiOrganizationId organization Kunta API id
+     * @param kuntaApiEventId event Kunta API id
+     * @param kuntaApiAttachmentId event image Kunta API id
+     * @param id MikkeliNyt id
+     * @param url MikkeliNyt URL
+     * @param name MikkeliNyt name
+     * @param description MikkeliNyt description
+     * @param city MikkeliNyt city
+     * @param address MikkeliNyt address 
+     * @param zip MikkeliNyt ZIP
+     * @param place MikkeliNyt place
+     * @param image MikkeliNyt image
+     * @param thumb MikkeliNyt thumb
+     * @param start MikkeliNyt start
+     * @param end MikkeliNyt end
+     * @return
+     */
+    @SuppressWarnings ("squid:S00107")
+    public Event mockEvent(String kuntaApiOrganizationId, String kuntaApiEventId, String kuntaApiAttachmentId, String id, String url, String name, String description, String city, String address, String zip, String place, String image, String thumb, OffsetDateTime start, OffsetDateTime end) {
+      String location = String.format("%s, %s", formatAddress(address), formatZip(zip));
+      
+      Event event = new Event();
+      event.setAddress(formatAddress(address));
+      event.setCity(formatCity(city));
+      event.setDescription(description);
+      event.setEnd(formatDateTime(end));
+      event.setId(id);
+      event.setImage(image);
+      event.setLocation(location);
+      event.setName(name);
+      event.setNiceDatetime(formatNiceDateRange(start, end));
+      event.setPlace(formatPlace(place));
+      event.setStart(formatDateTime(start));
+      event.setThumb(thumb);
+      event.setUrl(url);
+      event.setZip(formatZip(zip));
+      
+      Attachment imageAttachment = new Attachment();
+      imageAttachment.setContentType(TEST_EVENT_ATTACHMENT_TYPE);
+      imageAttachment.setSize(new Long(TEST_EVENT_ATTACHMENT_SIZE));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this "Long" constructor [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2129)